packages/p2p-v2/src/components/Modals/PaymentMethods/AddEditPaymentMethodErrorModal/AddEditPaymentMethodErrorModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import ReactModal from 'react-modal';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { customStyles } from '../../helpers';
+import './AddEditPaymentMethodErrorModal.scss';
+
+type TAddEditPaymentMethodErrorModalProps = {
+    errorMessage: string;
+    isOpen: boolean;
+    onConfirm: () => void;
+};
+
+const AddEditPaymentMethodErrorModal = ({ errorMessage, isOpen, onConfirm }: TAddEditPaymentMethodErrorModalProps) => {
+    useEffect(() => {
+        ReactModal.setAppElement('#v2_modal_root');
+    }, []);
+    // TODO: Remember to translate these strings
+    return (
+        <ReactModal
+            contentLabel="Something's not right"
+            isOpen={isOpen}
+            shouldCloseOnOverlayClick={false}
+            style={customStyles}
+        >
+            <div className='p2p-v2-add-edit-payment-methods-modal__wrapper'>
+                <Text color='prominent' weight='bold'>
+                    {"Something's not right"}
```

Comment:
@adrienne-deriv, the `'` character needs to be escaped here. Do you know a better way to do it? 🤔

Reply:
you can use option + shift + ], which is how we do it in some of the text's for p2p so it looks like this ’ instead of '

```suggestion
                    Something’s not right
````